src/Env.jsx

```diff
@@ -0,0 +1,87 @@
+
+        break;
+    }
+
+    return delta;
+}
+
+const handleEnvValue = (key, value, idx, onChange, additem, itemCount, companionField) => {
+    // Allow the input of KEY=VALUE separated value pairs for bulk import only if the other
+    // field is not empty.
+    if (value.includes('=') && !companionField) {
+        const parts = value.trim().split(" ");
+        let index = idx;
+        for (const part of parts) {
+            const [envKey, ...envVar] = part.split('=');
+            if (!envKey || !envVar) {
+                continue;
+            }
+
+            if (index !== idx) {
+                additem();
+            }
+            onChange(index, 'envKey', envKey);
+            onChange(index, 'envValue', envVar.join('='));
+            index++;
+        }
+    } else {
+        onChange(idx, key, value);
+    }
+};
+
+export const EnvVar = ({ id, item, onChange, idx, removeitem, additem, itemCount, validationFailed, onValidationChange }) =>
+    (
+        <Grid hasGutter id={id}>
+            <FormGroup className="pf-m-6-col-on-md" label={_("Key")} fieldId={id + "-key-address"} isRequired>
+                <TextInput id={id + "-key"}
+                       value={item.envKey || ''}
+                       validated={validationFailed?.envKey ? "error" : "default"}
+                       onBlur={() => onValidationChange(validateEnvVar(item, "envKey", validationFailed))}
+                       onFocus={() => utils.validationClear(validationFailed, "envKey", onValidationChange)}
+                       onChange={(_, value) => handleEnvValue('envKey', value, idx, onChange, additem, itemCount, item.envValue)} />
+                <FormHelper helperTextInvalid={validationFailed?.envKey} />
+            </FormGroup>
+            <FormGroup className="pf-m-6-col-on-md" label={_("Value")} fieldId={id + "-value-address"} isRequired>
+                <TextInput id={id + "-value"}
+                       value={item.envValue || ''}
+                       validated={validationFailed?.envValue ? "error" : "default"}
+                       onBlur={() => onValidationChange(validateEnvVar(item, "envValue", validationFailed))}
+                       onFocus={() => utils.validationClear(validationFailed, "envValue", onValidationChange)}
+                       onChange={(_, value) => handleEnvValue('envValue', value, idx, onChange, additem, itemCount, item.envKey)} />
+                <FormHelper helperTextInvalid={validationFailed?.envValue} />
+            </FormGroup>
+            <FormGroup className="pf-m-1-col-on-md remove-button-group">
+                <Button variant='plain'
+                    className="btn-close"
+                    id={id + "-btn-close"}
+                    size="sm"
+                    aria-label={_("Remove item")}
+                    icon={<TrashIcon />}
+                    onClick={() => removeitem(idx)} />
+            </FormGroup>
+        </Grid>
```

Comment:
These 29 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230907-093411-a9f37bd4-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)